tests/components/aseko_pool_live/test_config_flow.py

```diff
@@ -84,3 +117,83 @@ async def test_get_account_info_exceptions(
 
     assert result2["type"] == FlowResultType.FORM
     assert result2["errors"] == {"base": reason}
+
+
+async def test_async_step_reauth_form(hass: HomeAssistant) -> None:
+    """Test we get the form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_REAUTH, "entry_id": "test-id"}
+    )
+
+    assert result["type"] == FlowResultType.FORM
+    assert result["step_id"] == "reauth_confirm"
+    assert result["errors"] == {}
```

Comment:
```suggestion
```

Will be tested in `test_async_step_reauth_success`

Reply:
Removed